src/test/perl/test-caffileeditor_source.t

```diff
@@ -0,0 +1,66 @@
+#!/usr/bin/perl
+use strict;
+use warnings;
+use FindBin qw($Bin);
+use lib "$Bin/modules";
+use testapp;
+use CAF::FileEditor;
+use Test::More tests => 3;
+use Test::Quattor::Object;
+use Carp qw(confess);
+use File::Path;
+use File::Temp qw(tempfile);
+
+my $testdir = 'target/test/editor';
```

Comment:
the testdir has to be unique for the test, we cannot risk using directories with files from previous tests